dist/performance.js

```diff
@@ -2,6 +2,18 @@
 
 const response_bodies = $WPT_BODIES;
 
+function getRawHtmlDocument() {
+    let rawHtml;
+    if (response_bodies.length > 0) {
+        rawHtml = response_bodies[0].response_body;
```

Comment:
Do we have a confidence level for the 0th response body always being the raw HTML? I remember in the livestream you mentioned it's pretty likely - is there a way to test?

Reply:
Good catch, yes we [looked into it](https://github.com/HTTPArchive/custom-metrics/issues/15) and we can rely on the 0th element to be the document. 👍